lib/charms/mysql/v0/backups.py

```diff
@@ -48,7 +48,7 @@ def is_unit_blocked(self) -> bool:
 import datetime
 import logging
 import pathlib
-from typing import Dict, List, Tuple
+from typing import Dict, List, Optional, Tuple
 
 from charms.data_platform_libs.v0.s3 import S3Requirer
 from charms.mysql.v0.mysql import (
```

Comment:
You would need to update the patchlib of all the libs modified in this PR: backups and s3_helpers

Reply:
Fixed https://github.com/canonical/mysql-operator/pull/204/commits/d55dd494463f7aac062d2645faeeead48c0848fc